indexing-service/src/test/java/org/apache/druid/indexing/overlord/http/OverlordTest.java

```diff
@@ -270,19 +300,20 @@ public void testOverlordRun() throws Exception
     Assert.assertEquals(TASK_LOCATION, taskResponseObject.getLocation());
 
     // Simulate completion of task_1
-    taskCompletionCountDownLatches[Integer.parseInt(taskId_1)].countDown();
+    taskCompletionCountDownLatches.get(taskId_1).countDown();
     // Wait for taskQueue to handle success status of task_1
     waitForTaskStatus(taskId_1, TaskState.SUCCESS);
 
     // should return number of tasks which are not in running state
     response = overlordResource.getCompleteTasks(null, req);
-    Assert.assertEquals(2, (((List) response.getEntity()).size()));
+    Assert.assertEquals(4, (((List) response.getEntity()).size()));
```

Comment:
Why did we need to change an existing test?
I would advise adding a separate test for verifying the behaviour of such tasks where we failed to reacquire locks.